src/templates/userRetention.mjml

```diff
@@ -41,65 +38,90 @@
           <mj-text>
             {{/equals}}
           </mj-text>
-          <mj-spacer height="24px" />
-          <mj-text mj-class="color-grey-darker">
+          <mj-text>
             {{#greaterThan (length @root.articlesRecommended) 0}}
-              <mj-column>
-                {{#each @root.articlesRecommended}}
-                  <a class="article-digest link-black weight-medium article-recommended" href="{{@root.siteDomain}}/@-/-{{this.mediaHash}}">
-                    <span class="article-digest-title">{{this.title}}</span>
-                    <span class="article-digest-subtitle">{{this.displayName}}</span>
-                  </a>
-                {{/each}}
-              </mj-column>
+          </mj-text>
+          <mj-text mj-class="color-grey-darker">
```

Comment:
I think we should put the divider inside `articlesRecommended` block instead of L29-L31?